kinta-lib/src/main/kotlin/com/dailymotion/kinta/KintaEnv.kt

```diff
@@ -35,6 +40,8 @@ object KintaEnv {
         return System.getenv(key)
     }
 
+    fun getOrFail(env: Env) = getOrFail(env.name)
+
     fun getOrFail(key: String): String {
```

Comment:
Do we still need the `String` versions ?

Reply:
I have kept it for users who want to add their owes kinta env.
For exemple, our CI_BACKEND_TOKEN